### ISSUE TITLE
Grafana and Prometheus  implementation locally

### ISSUE THREAD

**Issue by povolev15** [hashgraph/hedera-services#4812]
implement Grafana and Prometheus locally.
Present some of the metrics

**Comment by povolev15** [comment]
waiting for the merge between Platform and Services repo in order to see where to place it if at all.

**Comment by povolev15** [comment]
@jsync-swirlds @nathanklick @kimbor 
Me and @hendrikebbers just tried to run local-node and than to start services locally and we got an error from services locally in parallel. 
The error we got from services as following:
 `2023-02-28 10:07:03.086 WARN  96   NettyGrpcServerManager - (Attempts=33) Still trying to start Netty on port 50211...Failed to bind to address 0.0.0.0/0.0.0.0:50211`

It is clear for us why the error happened but we are trying to find a solution just run the Prometheus and Grafana without starting the local node (from local-node repository).

cc: @nickpoorman

**Comment by nickpoorman** [comment]
Update: @hendrikebbers and @povolev15 are to sync up. Need to review PR

**Comment by nickpoorman** [comment]
Update: New PR to be opened, responding from comments from Nathan.

**Comment by povolev15** [comment]
Me and @hendrikebbers  had a discussion yesterday with @nathanklick, @dalvizu and @beeradb.
We will wait to @nathanklick and@beeradb to refactor the file structure to support the local environment in addition to the production environment as well.
The proposal Nathan suggested is here: https://www.notion.so/swirldslabs/Node-Software-Grafana-Resource-Repository-Structure-706712c8cb064dfcb28768bbd8fc6d84

**Comment by Neeharika-Sompalli** [comment]
Still waiting for devops

**Comment by netopyr** [comment]
@hendrikebbers @povolev15 Is this issue still relevant?

**Comment by povolev15** [comment]
I'm not sure.

**Comment by akugal** [comment]
There is already an [example app](https://github.com/akugal/hiero-consensus-node/tree/main/example-apps/swirlds-platform-base-example/docker) that has Docker containers for local observability.
